python/cuml/linear_model/mbsgd_classifier.pyx

```diff
@@ -18,11 +18,11 @@
 # distutils: language = c++
 # cython: embedsignature = True
 # cython: language_level = 3
-
+from cuml.common.base import Base
 from cuml.solvers import SGD
 
 
-class MBSGDClassifier:
+class MBSGDClassifier(Base):
```

Comment:
should add something in __init__ like: ```
 super().__init__(self, handle=handle, verbose=verbose)
```

and remove the `self.handle = ` bit since the super class will do it.

Reply:
Right, makes sense, on it!

Reply:
For consistency the line typically should be `super(MBSGDClassifier, self).__init__(handle, verbose)` also might prove useful to do it that way in the future